src/Builder/Pipeline.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace MongoDB\Builder;
+
+use MongoDB\Builder\Stage\Stage;
+use MongoDB\Exception\InvalidArgumentException;
+
+use function array_is_list;
+
+class Pipeline
+{
+    public array $stages;
+
+    public function __construct(
+        Stage ...$stages
```

Comment:
`Pipelines` must be accepted and merged with stages. [PHPLIB-1264](https://jira.mongodb.org/browse/PHPLIB-1264)

Reply:
Was that an original requirement of the [technical design](https://docs.google.com/document/d/1vFWXB_6nnYH6p136alKDnIHekW-JeNA7nH3vWSriDtY/edit)? If so, I'm having trouble finding it in the document.

Is this syntactic sugar preferable to adding a method on a Pipeline to return its Stages as a list, and then merging those with a splat operator (e.g. `...$pipeline->getStages(),`)?

Reply:
Will track in [PHPLIB-1264](https://jira.mongodb.org/browse/PHPLIB-1264).